CriteoPublisherSdk/Sources/Public/CRUserData.h

```diff
@@ -0,0 +1,30 @@
+//
+//  CRUserData.h
+//  CriteoPublisherSdk
+//
+//  Copyright © 2018-2020 Criteo. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface CRUserData : NSObject
+
++ (CRUserData *)dataWithDictionary:(NSDictionary<NSString *, id> *)dictionary;
```

Comment:
I would suggest to expose the counterpart instance method `initWithDictionary` as well, as it is often what the developers are looking for and also because this is what I see in the style of most of the SDK.
I realise this applies as well to the `CRContextData` class.

Reply:
Done